### DISCUSSION TITLE
Onboarding for contributing

### DISCUSSION THREAD

**Discussion by naftalibeder** [adbar/trafilatura#154] [Q&A]
@adbar Would it be possible to write up some advice for would-be contributors? I’m finding little accuracy bugs here and there, and I’d love to help out, but would get a lot out of a few notes if you have the time.

Anything would be useful, but particularly describing the flow you follow in the common scenario where the extracted text for a site is too broad or narrow. I have a few of these I’d like to pursue, but I’m at a bit of a loss for where to start. 

Thanks for the wonderful library!

**Comment by adbar** [comment]
Hi @naftalibeder, thanks for your feedback!

**Option 1**

Did you check out this thread? I leave it open for people to list smaller issues. Maybe it wasn't quite clear so I just updated the description:
https://github.com/adbar/trafilatura/issues/4
If for some reason the main text of a page is missed completely then you could file single issues for those pages.

**Option 2**

You can also file separate threads if you think the issues are generic or severe. Here are examples of issues that have been filed in the past:
- Particular issue with missed title: https://github.com/adbar/trafilatura/issues/71
- Generic issue with formatting of lists: https://github.com/adbar/trafilatura/issues/58
- Problem with links: https://github.com/adbar/trafilatura/issues/147

In any case, don't hesitate to report issues, I can always classify and regroup them if necessary.
